src/components/molecules/Footer/index.jsx

```diff
@@ -146,12 +148,10 @@ class Footer extends React.PureComponent {
               <p className="mt-2">Hochwertige und individuelle Lösungen</p>
               <hr />
               <h4>Dark Mode</h4>
-              <small className="text-muted red-text">Bald verfügbar</small>
               <MDBSwitch
                 className="switch-red mb-3"
-                checked={this.props.mode}
+                checked={this.props.darkMode}
```

Comment:
This is not a stateful switch. Implement a `componentDidMount` where you `this.setState({darkMode: this.props.darkMode ? true : false})` and use the state on the `checked` param.